packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../devtools.dart' as devtools;
+import '../../flutter/common_widgets.dart';
+import '../../flutter/screen.dart';
+import '../../version.dart';
+import '../info_controller.dart';
+
+class InfoScreen extends Screen {
+  const InfoScreen() : super('Info');
+
+  @override
+  Widget build(BuildContext context) => InfoScreenBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      icon: Icon(Icons.info),
+      text: 'Info',
+    );
+  }
+
+  /// The key to identify the flag list view
+  @visibleForTesting
+  static const Key flagListKey = Key('Info Screen Flag List');
+
+  /// The key to identify the flutter version view.
+  @visibleForTesting
+  static const Key flutterVersionKey = Key('Info Screen Flutter Version');
+}
+
+class InfoScreenBody extends StatefulWidget {
+  @override
+  _InfoScreenBodyState createState() => _InfoScreenBodyState();
+}
+
+class _InfoScreenBodyState extends State<InfoScreenBody> {
+  FlutterVersion _flutterVersion;
+  FlagList _flagList;
```

Comment:
Nit: new line between class members. Feel free to push back if this is dart style and not flutter

Reply:
I think in cases where there's no documentation, it's not too important either way.  I'm happy to leave the extra space.